server/routes/contacts/manage/relationship/manageNextOfKinContactController.ts

```diff
@@ -0,0 +1,48 @@
+import { Request, Response } from 'express'
+import { PageHandler } from '../../../../interfaces/pageHandler'
+import { Page } from '../../../../services/auditService'
+import { ContactsService } from '../../../../services'
+import Contact = contactsApiClientTypes.Contact
+import UpdateRelationshipRequest = contactsApiClientTypes.UpdateRelationshipRequest
+
+export default class ManageNextOfKinContactController implements PageHandler {
+  constructor(private readonly contactsService: ContactsService) {}
+
+  public PAGE_NAME = Page.MANAGE_CONTACT_EDIT_NEXT_OF_KIN_STATUS_PAGE
+
+  GET = async (req: Request<{ contactId?: string; prisonerContactId?: string }>, res: Response): Promise<void> => {
+    const { contactId, prisonerContactId } = req.params
+    const { prisonerDetails, user } = res.locals
+
+    const contact: Contact = await this.contactsService.getContact(parseInt(contactId, 10), user)
+    const relationship = await this.contactsService.getPrisonerContactRelationship(Number(prisonerContactId), user)
+
+    return res.render('pages/contacts/manage/contactDetails/manageNextOfKinContactStatus', {
+      contact,
+      prisonerContactId,
+      relationship,
+      prisonerDetails,
+    })
+  }
+
+  POST = async (
+    req: Request<{ contactId: string; prisonerNumber: string; prisonerContactId: string }>,
+    res: Response,
+  ): Promise<void> => {
+    const { user } = res.locals
+    const { contactId, prisonerNumber, prisonerContactId } = req.params
+    const request: UpdateRelationshipRequest = {
+      isNextOfKin: req.body.nextOfKinStatus === 'YES',
+      updatedBy: user.userId,
```

Comment:
Should be username, see thread in dev channel

Reply:
> updated to use username